backend/seeds/00800_matches.js

```diff
@@ -0,0 +1,25 @@
+const { tableStatusUpdates } = require('../src/models/statusUpdate')
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex(tableStatusUpdates)
+    .del()
+    .then(function () {
+      // Inserts seed entries
+      return knex(tableStatusUpdates).insert([
```

Comment:
I think we should insert a TeacherID too. In fact, we could make StudentID and TeacherID non-nullable, since I also think it's illegal to have a match record without either a student or a teacher.

Reply:
`TeacherID` and `StudentID` are now non-nullable.